### DISCUSSION TITLE
Block displayed in the last page that stick the bottom

### DISCUSSION THREAD

**Discussion by nboukeffa** [gotenberg/gotenberg#486] [Q&A]
Hello here! 

I'm trying to find a way to display a block that sticks to the bottom of the page only on the last page. 

I tried different CSS approaches (fixed, absolute, ...) but none of them seems correct.

Did someone know how to do that? Here is an example of what I want to do:

![Untitled](https://user-images.githubusercontent.com/154042/184324748-fca2ac1c-56e3-4587-acbd-4acf8743e3b4.png)

Thanks for your help!

**Comment by gulien** [comment]
Hello @nboukeffa,

https://pagedjs.org/ might help you to solve this.

**Comment by jsheehan-tic** [comment]
Did this go anywhere? Facing a similar issue atm.
